corehq/apps/saved_reports/management/commands/daylight_savings.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime, timedelta
+from django.core.management import BaseCommand
+
+from corehq.apps.saved_reports.models import ReportNotification
+
+
+def get_domains(path):
+    with open(path, 'r') as f:
+        domains = f.readlines()
+
+    return [domain.strip() for domain in domains]
+
+
+def get_reports_by_domain(domain):
+    key = [domain]
+    reports = ReportNotification.view('reportconfig/user_notifications',
+        reduce=False, include_docs=True, startkey=key, endkey=key + [{}])
+    return reports
+
+
+DAYS_IN_WEEK = 7
+
+
+def adjust_report(report, forward=False):
+    day = report.day + 1 if report.interval == 'weekly' else report.day  # account for 0-indexed days
+    trigger_time = datetime.now().replace(hour=report.hour, minute=report.minute, day=day)
+
+    if forward:
+        trigger_time += timedelta(hours=1)
+    else:
+        trigger_time -= timedelta(hours=1)
+
+    report.hour = trigger_time.hour
+    if report.interval == 'weekly':
+        report.day = (trigger_time.day - 1) % DAYS_IN_WEEK
+    elif report.interval == 'monthly':
+        report.day = trigger_time.day
+
+    return report
+
+
+class Command(BaseCommand):
```

Comment:
nit: I like when functions and classes are ordered using the [newspaper metaphor](https://github.com/jbarroso/clean-code#the-newspaper-metaphor), where the things a new reader would most likely want to see are at the top and utility functions are toward the bottom. This module appears to have the opposite order.